guidelines.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+title: Package Guidelines
+description: Guidelines for Pangeo packages to follow
+---
+
+Our vision for the Pangeo project is an ecosystem of mutually compatible
+Geoscience python packages which follow open-source best practices.
+These practices are well established across the scientific python community.
+
+
+### General Best Practices for Open Source
+
+1. Open source license
+1. Community guidelines (provide example)
+1. Version control of source code (for example, on [github](http://github.org))
+1. Thorough test coverage and continuous integration of testing
+1. Comprehensive Documentation
+
+
+### Best Practices for Pangeo Projects
+
+To address the needs of geoscience researchers, we have developed some
+additional recommendations.
+
+1. _Solve a general problem:_ packages should solve a general problem
+that is encountered by broad groups of researchers.
+1. _Clearly defined scope:_ packages should have a clear and relatively
+narrow scope.
```

Comment:
Given that this sounds superficially like the opposite of the preceding point, maybe worth adding a sentence or two clarifying that these are not mutually exclusive: we're looking for the Goldilocks packages that are sufficiently-but-not-excessively broad.  

Another similar point: a package being small doesn't necessarily mean it has a well-defined scope, and vice versa.